gprotorch/dataloader/mol_prop.py

```diff
@@ -3,16 +3,19 @@
 molecular property prediction datasets.
```

Comment:
Perhaps we could rename this module `molecule_loader` in line with the new `reaction_loader` module? What do you think @leojklarner ?

Reply:
Yes, that would clean things up. I can take care of that when doing the GProTorch --> GAUCHE renaming.